naga/tests/in/spv/binding-arrays.dynamic.spvasm

```diff
@@ -1,3 +1,6 @@
+; SPIR-V
```

Comment:
If this little header isn't present, than we hit https://github.com/KhronosGroup/SPIRV-Tools/issues/5909 on the SDK I currently have installed.

Reply:
...caaan we include this as a comment, instead of a buried PR comment? 😀

Reply:
I can, I just feel weird putting the same comment on every single file